dandi/pynwb_utils.py

```diff
@@ -315,7 +315,8 @@ def rename_nwb_external_files(metadata: List[dict], dandiset_path: str) -> None:
                         ext_file_dict["external_files_renamed"],
                     )
                 ):
-                    container.external_file[no] = str(name_new)
+                    if not any([str(name_old).startswith(i) for i in ["http", "ftp"]]):
```

Comment:
2nd encounter -- thus better add a short function `is_url` or alike implementing that , and reuse in both spots

Reply:
actually -- we already have `dandi.utils.is_url` which you might just want to extend with `ftp://`. Note that it also takes `dandi:` ones as urls, imho legit

Reply:
thanks for pointing that out, I have now used that function instead by adding the "ftp" check to that